packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -14,11 +14,1064 @@ The Okta Administrator Roles API provides operations to manage administrative ro
+
+Get a paginated list of resource sets
+
+#### Response parameters
+
+Paginated list of [resource sets](#resource-set-object)
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets"
+```
+
+#### Response example
+
+```json
+{
+  "resource-sets": [
+    {
+      "id": "iamoJDFKaJxGIr0oamd9g"
+      "label": "SF-IT-1",
+      "description": "...",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+        },
+        "resources": {
+          "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+        },
+        "bindings": {
+          "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+        }
+      },
+      ...
+    ],
+      "_links": {
+        "next": {
+          "href": "http://${yourOktaDomain}/api/v1/iam/resource-sets?after=iamoJDFKaJxGIr0oamd9g"
+        }
+      }
+    }
+  }
+}
+```
+
+### Update resource set
+<ApiOperation method="put" url="/api/v1/iam/resource-sets/${resourceSetId}" />
+
+Update label and description of a resource set
+
+#### Request parameters
+
+| Parameter     | Description                               | Param Type   | DataType     | Required |
+| :------------ | :---------------------------------------- | :----------- | :----------- | :------- |
+| resourceSetId | Unique id of the resource set             | URL          | String       | TRUE     |
+| label         | New unique name given to the resource set | Body         | String       | TRUE     |
+| description   | New description of the the resource set   | Body         | String       | TRUE     |
+
+#### Response parameters
+
+Updated [resource set](#resource-set-object)
+
+#### Request example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "label": "SF-IT-Staff",
+      "description": "Staff in the IT department of San Francisco"
+    }' "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+```
+
+#### Response example
+
+```json
+{
+  "id": "iamoJDFKaJxGIr0oamd9g",
+  "label": "SF-IT-Staff",
+  "description": "Staff in the IT department of San Francisco",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    },
+    "resources": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+    },
+    "bindings": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+    }
+  }
+}
+```
+
+### Delete resource set
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}" />
+
+Deletes a resource set
```

Comment:
worth some treatment re what happens to bindings which were using this resource set (i.e. they are deleted as well)?

Reply:
updated